content/vagrant-local-for-k8s.md

```diff
@@ -0,0 +1,193 @@
+    # via 127.0.0.1 to disable public access
+    # config.vm.network "forwarded_port", guest: 80, host: 8080, host_ip: "127.0.0.1"
+
+    # Create a private network, which allows host-only access to the machine
+    # using a specific IP.
+    config.vm.network "private_network", ip: "192.168.33.10"
+
+    # Create a public network, which generally matched to bridged network.
+    # Bridged networks make the machine appear as another physical device on
+    # your network.
+    # config.vm.network "public_network"
+
+    # Share an additional folder to the guest VM.
+    # config.vm.synced_folder "../data", "/vagrant_data"
+
+    # Provider-specific configuration so you can fine-tune various
+    # backing providers for Vagrant. These expose provider-specific options.
+    # Example for VirtualBox:
+    #
+    # config.vm.provider "virtualbox" do |vb|
+    #   # Display the VirtualBox GUI when booting the machine
+    #   vb.gui = true
+    #
+    #   # Customize the amount of memory on the VM:
+    #   vb.memory = "1024"
+    # end
+
+    # Enable provisioning with a shell script. Additional provisioners such as
+    # Ansible, Chef, Docker, Puppet and Salt are also available.
+    # config.vm.provision "shell", inline: <<-SHELL
+    #   apt-get update
+    #   apt-get install -y apache2
+    # SHELL
+    end
+
+Like with any VM, there's basically 5 important parts here: 
+- base image declaration - this defines the base VM image you want to start from. I'm choosing my favorite Linux distribution, openSUSE Leap, in it's current version 15.2. More on Vagrant base boxes later. 
+- network config - how your vagrant box can be reached. As you can see, you can choose between exposing specific ports (which can be limited to being accessible only from certain IP addresses), exposing a host-only IP, or bridging the guest's virtual network interface to the host's external network so that it becomes visible to other machines in the same network. The latter needs to be handled with care obviously - don't forget to turn off your VMs before heading off to your next conference (whenever that will be). I'm going with a private IP for now. 
+- Shared folders for data exchange between guest and host (note that by default, vagrant mounts the directory where your vagrantfile is to /vagrant in the guest, which is perfectly fine for copying the occasional config file around and similar things). 
+- provider config (e.g. hypervisor settings - here's where you for example define how much memory your VM should have)
+- provisioning (e.g. install security updates or extra packages post-install). The example shown here uses the so-called shell provisioner, but you can also use proper provisioning languages like Ansible, Salt, etc.
+
+So, now that we have put together a basic configuration, let's take our new box out for a first spin. All we need to do for this is to run a simple `vagrant up` in the vagrant file directory. 
+
+    > vagrant up
```

Comment:
I wonder how possible it would be to get ascii-cinema available on the suse blog?